airbrake/notifier.py

```diff
@@ -138,6 +138,11 @@ def __init__(self, project_id=None, api_key=None, host=None, timeout=None,
         self.blacklist_keys = config.get("blacklist_keys", [])
         self.filter_chain = [self.filter_whitelist, self.filter_blacklist]
 
+        self.send_uncaught_exc = config.get("send_uncaught_exc", True)
+        if self.send_uncaught_exc:
+            self.excepthook = sys.excepthook
```

Comment:
`sys.excepthook` is invoked whenever an exception is raised __but__ uncaught :+1: